### ISSUE TITLE
multiple responses to direct requests

### ISSUE THREAD

**Issue by jonorthwash** [apertium/phenny#430]
When begiak is asked to provide someone with information, it responds with multiple messages:
```
(10:38:57) firespeaker: .awik APy → Hous
(10:38:59) begiak: Hous, "Apertium-APy stands for "Apertium API in Python"" - http://wiki.apertium.org/wiki/APy
(10:39:00) begiak: Can't find anything in the Apertium Wiki for "APy → Hous".
```
```
(10:47:57) firespeaker: .w Apertium → Flammie
(10:47:58) begiak: Perhaps you meant 'apertum flamme'?
(10:47:59) begiak: Perhaps you meant 'ser'?
(10:48:02) begiak: Flammie: ser — noun: 1. used in some fantasy novels: An address or courtesy title to any person, especially if their gender and/or form of address are unknown, 2. being — verb: 1. to be, 2. to be
```

However, sometimes it's fine:
```
(10:44:38) firespeaker: .wik Apertium
(10:44:39) begiak: "Apertium is a free/open-source rule-based machine translation platform" - https://en.wikipedia.org/wiki/Apertium
```
```
(10:44:54) firespeaker: .awik Apertium
(10:44:56) begiak: "This Wiki is all about Apertium, a free-software / open-source machine translation system for closely (and not so closely) related languages" - http://wiki.apertium.org/wiki/Apertium
```

**Comment by Androbin** [comment]
Yeah, it's an issue with pointing duplicates.
I think I've fixed that in cd73a90a39e6dd963f69b988ea0dab0cf38a8fc7.

**Comment by Androbin** [comment]
There was an issue with the non-pointing regexes. Fixed in fdf32e7814b4d422b9446eb3e1b47fbb71ac4457.
